### DISCUSSION TITLE
Electron + parcel: Failed to load resource: net::ERR_FILE_NOT_FOUND

### DISCUSSION THREAD

**Discussion by jogibs** [parcel-bundler/parcel#8080] [Q&A] [answered]
I'm building an Electron + Parcel app, but I've encountered an issue when opening the app. The app itself gets build and I see the html template, but it won't load the script, it says: Failed to load resource: net::ERR_FILE_NOT_FOUND

main.js
```
const mainWindow = new BrowserWindow({
  width: 800,
  height: 600,
  show: false,
  webPreferences: {
    preload: join(__dirname, 'preload.js'),
  },
});
mainWindow.loadURL(join(__dirname, 'index.html'));
```

The final result that I get from parcel is a minified html file, which looks something like:
```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="theme-color" content="#000000">
  <title>App</title>
  <script src="/index.8ce482f2.js"></script>
</head>
<body>
...
</body>
</html>
```

Now, I do roughly understand the issue - it's probably because the `/index.8ce482f2.js` gets evaluated into `file:\\C:\index.8ce482f2.js` once it opens the app, but I'm not really sure how to fix it. The obvious answer would be to just remove the forward slash, but that's rather tricky, because that's the HTML that gets generated by parcel every run.

Is there any better solution or setting that I can enable in electron and/or parcel for it to allow forward slashes or that the forward slashes get removed on build time?

**Comment by jogibs** [comment, marked as answer]
Solved it by adding a `--public-url .` flag to parcel.

**Comment by ec2-learn-instant** [comment]
Where to add this "--public-url"?
